code/modules/mob/living/carbon/alien/humanoid/update_icons.dm

```diff
@@ -42,6 +42,8 @@
 			alt_icon = old_icon
 		pixel_x = get_standard_pixel_x_offset(lying)
 		pixel_y = get_standard_pixel_y_offset(lying)
+	update_inv_hands()
+	update_inv_handcuffed()

```

Comment:
should probably put this one behind a handcuffed check to avoid unnecessary overlay updates